add more objects to showcase CI

tdeneke opened the pull request.

lvarin commented:
Looks nice, but could you add to README a mention of the 3 objects that you are also adding? And a short description of what they do and we added them?

tdeneke commented:
> Looks nice, but could you add to README a mention of the 3 objects that you are also adding? And a short description of what they do and we added them?

Added.
